httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/StringBody.java

```diff
@@ -70,13 +70,7 @@ public Reader getReader() {
     @Override
     public void writeTo(final OutputStream out) throws IOException {
         Args.notNull(out, "Output stream");
-        final InputStream in = new ByteArrayInputStream(this.content);
```

Comment:
I think the `InputStream` import can be removed.

Reply:
oh good catch! will fix